File: app/src/main/java/com/mediaplayer/listeners/PopUpOnTouchListener.java
package com.mediaplayer.listeners;


import android.view.MotionEvent;
import android.view.View;
import android.widget.PopupWindow;

public class PopUpOnTouchListener implements View.OnTouchListener {

    public PopupWindow pw;

    @Override
    public boolean onTouch(View v, MotionEvent event) {
        pw.dismiss();
        return true;
    }
}

File: app/src/main/java/com/mediaplayer/listeners/PopUpOnTouchListener.kt
package com.mediaplayer.listeners


import android.annotation.SuppressLint
import android.view.MotionEvent
import android.view.View
import android.widget.PopupWindow

class PopUpOnTouchListener : View.OnTouchListener {

    var pw: PopupWindow? = null

    @SuppressLint("ClickableViewAccessibility")
    override fun onTouch(v: View, event: MotionEvent): Boolean {
        pw!!.dismiss()
        return true
    }
}
